src/html-media-source.js

```diff
@@ -211,7 +216,7 @@ export default class HtmlMediaSource extends videojs.EventTarget {
     // Automatically disable the audio on the first source buffer if
     // a second source buffer is ever created
     if (this.sourceBuffers.length !== 0) {
-      this.sourceBuffers[0].disableAudio();
+      this.sourceBuffers[0].audioDisabled_ = true;
```

Comment:
for the test in html called "does not wrap mp4 source buffers" it was throwing an error, since native source buffers will not have a `disableAudio()` function, so I changed it to just change to property which can be created but not used on native source buffers. We may even want to change all `disableAudio` and `enableAudio` calls in html-media-source to statements like this?